WcaOnRails/app/javascript/polyfills/index.js

```diff
@@ -0,0 +1,50 @@
+import 'whatwg-fetch';
+import Promise from 'promise-polyfill';
```

Comment:
I don't quite get why `Promise` and `#find` polyfills go here. Both are defined in `babel-polyfill` - that we already have (that should be imported here anyway, because I think it's not used at the moment).

Reply:
Ah, this stuff is here to make PhantomJS happy. I didn't know about `babel-polyfill`! I'll give it a shot and see if PhantomJS is happy.